lib/vagrant-cloudstack/action/run_instance.rb

```diff
@@ -34,7 +35,7 @@ def call(env)
           @network          = CloudstackResource.new(domain_config.network_id, domain_config.network_name, 'network')
           @service_offering = CloudstackResource.new(domain_config.service_offering_id, domain_config.service_offering_name, 'service_offering')
           @disk_offering    = CloudstackResource.new(domain_config.disk_offering_id, domain_config.disk_offering_name, 'disk_offering')
-          @template         = CloudstackResource.new(domain_config.template_id, domain_config.template_name, 'template')
+          @template         = CloudstackResource.new(domain_config.template_id, domain_config.template_name || env[:machine].config.vm.box, 'template')
```

Comment:
Apparently not as this works on my laptop :wink:
I always specify the box_url to the "dummy" box in the Vagrant file:
`config.vm.box_url = "https://github.com/schubergphilis/vagrant-cloudstack/raw/master/dummy.box"`
this way I don't have to separately download the box each time/sometimes/first time, and Vagrant probably directly uses the url to get/check the dummy box.

On the other hand if I have a succeful run and I remove the box (`vagrant box remove dummy`) and url reference, it still works :confused: 

But now you mention it, I think that specifying the box_url in the Vagrant file might be a requirement for this to (allways) work.